bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3247,7 +3247,7 @@ void ReplicaImp::onNewView(const std::vector<PrePrepareMsg *> &prePreparesForNew
   metric_current_active_view_.Get().Set(getCurrentView());
   metric_sent_replica_asks_to_leave_view_msg_.Get().Set(0);
 
-  onViewNumCallbacks_.invokeAll(getCurrentView());
+  onViewNumCallbacks_.invokeAll(isCurrentPrimary());
```

Comment:
It is correct that currently the other registered method doesn't use the parameter, but I think we will need to either change the callbacks registry to work with boolean or to have a comment here